pages/ecosystem/index.vue

```diff
@@ -129,7 +128,31 @@ useHead({
   ]
 })
 
-function getTestRows (member: any): void {
+const { data: members } = useLazyAsyncData(
+  'fetch-members',
+  async () => await import('~/content/ecosystem/members.json')
+)
+
+const { data: tiers } = useLazyAsyncData(
+  'fetch-tiers',
+  async () => await import('~/content/ecosystem/tiers.json')
+)
+
+const tierFilters = ref([] as string[])
```

Comment:
This type assertion can be replaced with a regular type.

```suggestion
const tierFilters = ref<string[]>([])
```